src/Data/CallbackRequest/callbackRequestParamsFallback.ts

```diff
@@ -9,6 +9,9 @@ async function attributes(): Promise<DataAttributeDefinitions> {
   }
 }
 
-export function localStorageCallbackRequestDataClass() {
-  return provideLocalStorageDataClass('CallbackRequest', { attributes })
+export function callbackRequestParamsFallback() {
```

Comment:
* [ ] FYI Naming: `*DataClass*` was clearer than `*Params*`. Params are just a wildcard and can mean anything, while a data class is meant to be passed to `provideDataClass`. How about `*FallbackDataClass` or `*DataClassFallback`?